index.html

```diff
@@ -142,13 +142,33 @@ <h6>Skills</h6>
 <figure id="formimage"></figure>
 <div class="wrapper footer">
 <div><p id=footertext>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p></div>
-<div class="form">
-<form action="https://formspree.io/f/meqwblog" method="post">
-      <input type="text" id="name" name="user_name" placeholder="Full Name" maxlength="30" required>
-      <input type="email" id="mail" name="user_email" placeholder="Email Address" pattern="[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}" required>
-      <textarea id="msg" name="user_message" placeholder="Hey I'd like to connect to..." maxlength="500" required></textarea>
-      <button id="formbutton" type="submit" name="submit" class="button">Get in touch</button>
-</form>
+
+<div class="form-container">
+  <form id="form" class="form" action="https://formspree.io/f/meqwblog" method="post">
```

Comment:
- [ ] After checking that your validation is OK, the form should be sent but It looks like the data does not be sent . Please fix it 👍 

![Capture](https://github.com/dvcorreia1984/portfolio/assets/6615394/89dadd8a-0904-450f-9ba2-374516e521c6)